app/cli/Main.hs

```diff
@@ -146,13 +169,17 @@ exec args
             (fromText @(Passphrase "encryption"))
             "Please enter a passphrase: \n\
             \(Enter a blank line if you do not wish to use a passphrase.)"
-        print
-            ( poolGap :: AddressPoolGap
-            , name :: WalletName
-            , mnemonic
-            , sndFactor
-            , passphrase
-            )
+        let walletData =
+                WalletPostData
+                    (Just $ ApiT poolGap)
+                    (ApiMnemonicT (mnemonic, ["todo"]))
```

Comment:
@KtorZ 
I don't have access to raw mnemonic sentance for `ApiMnemonicT`. Would we prefer to:
 1) Add `ToMnemonic` class similar to `FromMnemonic` that would convert `Passphrase` back to `[Text]`. This would involve implementing `ToMnemonic` typeclass and few instances in https://github.com/input-output-hk/cardano-wallet/blob/f54e90fcba9c60e780c0b4c710f335f8149c158d/src/Cardano/Wallet/Primitive/AddressDerivation.hs#L267 .
 
 2) Modify `getRequiredSensitiveValue` and `getOptionalSensitiveValue` to return tuple `IO (a, Text)` instead of current `IO a`

Reply:
After playing with option 2) I believe implementing 1) is a bit more work but safer to have

Reply:
> Passphrase back to [Text]

This isn't possible :no_mouth: ... We have to go for 2), although, returning `a` is not really necessary here. We may simply try to parse the mnemonic, and in case of success, return the raw version as a [Text]

Reply:
> This isn't possible

hm - I had a quick look at the `FromMnemonic` instance https://github.com/input-output-hk/cardano-wallet/blob/f54e90fcba9c60e780c0b4c710f335f8149c158d/src/Cardano/Wallet/Primitive/AddressDerivation.hs#L322 and my intuition was: well we just have to do `entropyToBytes $ mnemonicToEntropy m` in reverse - so something like `toMnemonic . toEntropy`. But maybe there is hidden complexity that I have overlooked.

Reply:
To go from mnemonic words to entropy, there are actually some hash going on. So it's not reversible :/